.github/workflows/limit-pr-lines-added.yml

```diff
@@ -73,11 +73,11 @@ jobs:
             SCHEMA_FILE=true
           fi
 
-          # Calculate total lines changed for the file
-          TOTAL=$((added + removed))
-          TOTAL_LINED_ADDED=$((TOTAL_LINED_ADDED + added))
+          # Calculate total lines added for the file
+          TOTAL=$((added))
```

Comment:
are we going to use `removed` ? may be just a log it as information?

Reply:
I added some logging of the removed lines in the latest commit.